src/ducks/balance/components/AccountsList.jsx

```diff
@@ -18,7 +21,13 @@ class AccountsList extends React.PureComponent {
   }
 
   render() {
-    const { accounts, warningLimit } = this.props
+    const {
+      accounts,
+      warningLimit,
+      switches,
+      switchesIdPrefix,
```

Comment:
The switchesIdPrefix directly maps to the groups, I think it would be simpler not to introduce a new concept here (switch prefix id) and pass simply the group.